.idea/codeStyles/Project.xml

```diff
@@ -1,6 +1,22 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
+        <value>
+          <package name="java.util" alias="false" withSubpackages="false" />
```

Comment:
We dont have to push .idea file in git. So we should add them to .gitignore. Meanwhile remove the current ones

Reply:
Resolved